### DISCUSSION TITLE
Introduction of your theme

### DISCUSSION THREAD

**Discussion by emisjerry** [ebullient/obsidian-theme-ebullientworks#9] [Show and tell]
Hi, i just want you know that i've made an introduction of your theme in Chinese, wish there'll be more users use it: https://youtu.be/AtibeL0tceo

Happy new year! ❤️

**Comment by ebullient** [comment]
Thank you so much! I've moved this to discussions, as it isn't an "issue" that should be closed. ;) I did use google translate some of the comments, and they were nice, too!

Please let me know if anything doesn't look right with Live Preview. I tend to stick with source mode, but I've been trying to keep up with Live Preview (I change a lot less than some themes, so I wasn't as impacted as other themes have been).
